purchases-js-hybrid-mappings/api-report/api.json

```diff
@@ -182,7 +182,7 @@
               "text": "declare class PurchasesCommon "
             }
           ],
-          "fileUrlPath": "src/index.ts",
+          "fileUrlPath": "dist/index.d.ts",

```

Comment:
Not sure if this change is a problem

Reply:
Well this change itself it’s not a problem since we just use this to diff the public API… I do wonder why it changed though 🤔